oshmem/mca/sshmem/verbs/sshmem_verbs_component.c

```diff
@@ -100,6 +101,11 @@ verbs_runtime_query(mca_base_module_t **module,
     *priority = 0;
     *module = NULL;
 
+    /* If fork support is requested, try to enable it */
+    if (OSHMEM_SUCCESS != (rc = opal_common_verbs_fork_test())) {
+        return OSHMEM_ERROR;
```

Comment:
I tried to keep the "verbosity behaviour" of every component and print out an additional message only if this is the way the component handeled errors from function calls.
Also, the opal_common_verbs_fork_test function would print out an error message if something went wrong there anyway.